monkey/monkey_island/cc/setup/island_config_options.py

```diff
@@ -1,57 +1,38 @@
 from __future__ import annotations
 
-from types import MappingProxyType as ImmutableMapping
-from typing import Mapping
+from pathlib import Path
+from typing import Annotated
+
+from monkeytypes import InfectionMonkeyBaseModel
+from pydantic import BeforeValidator, Field
 
 from common.utils.file_utils import expand_path
 from monkey_island.cc.server_utils.consts import (
     DEFAULT_CRT_PATH,
     DEFAULT_DATA_DIR,
     DEFAULT_KEY_PATH,
     DEFAULT_LOG_LEVEL,
+    DEFAULT_NODE_PORT,
     DEFAULT_START_MONGO_DB,
```

Comment:
These constants only get used in this file. We should move their definitions to here and "define things close to where they're used."